index.html

```diff
@@ -69,7 +69,7 @@ <h3 id="participate" class="translatable">Participate</h3>
 			<ul></ul>
 		</div>
 		<div class="tosdr">
-			<p class="translatable label" style="display:block"><abbr title='Terms of service ratings provided by https://tosdr.org'>TOS rating</abbr></p>
+			<p class="label" style="display:block"><abbr class="translatable" title='Terms of service ratings provided by https://tosdr.org'>TOS rating</abbr></p>
```

Comment:
It looks like no matter where you put the class, it would only apply to the content and not for the `title` attribute. See [comment below](#issuecomment-307375139)